README.md

```diff
@@ -95,12 +95,24 @@ containerized sandbox environment.
 
 ## Supported Protocol Versions
 
-**ZDM Proxy supports protocol versions v3, v4, DSE_V1 and DSE_V2.**
+**ZDM Proxy supports protocol versions v2, v3, v4, DSE_V1 and DSE_V2.**
 
 It technically doesn't support v5, but handles protocol negotiation so that the client application properly downgrades
 the protocol version to v4 if v5 is requested. This means that any client application using a recent driver that supports
 protocol version v5 can be migrated using the ZDM Proxy (as long as it does not use v5-specific functionality).
 
+ZDM Proxy requires origin and target clusters to have at least one protocol version in common. It is therefore not feasible
+to configure Apache Cassandra 2.0 as origin and 3.x / 4.x as target. Below table displays protocol versions supported by
+various C* versions:
+
+| Apache Cassandra | Protocol Version |
+|------------------|------------------|
+| 2.0              | V2               |
+| 2.1              | V2, V3           |
+| 2.2              | V3, V4           |
+| 3.x              | V3, V4           |
```

Comment:
`2.2` supports V4? Are you sure? `3.x` doesn't support V2? Can you tell me the source for this? All this time I thought otherwise